src/packages/website/templates/website/bower.json

```diff
@@ -24,6 +25,7 @@
     }
   },
   "resolutions": {
-    "angular": "1.5.x"
+    "angular": "1.5.x",
+    "angular-animate": "1.5.9"
```

Comment:
so sorry for so long time no reaction... 
I had was losing myself in a big project for that long time.
I'm back ^-^